opentargets_pharmgkb/validation.py

```diff
@@ -0,0 +1,33 @@
+import json
+import logging
+
+import jsonschema
+import requests
+
+logging.basicConfig()
+logger = logging.getLogger(__package__)
+logger.setLevel(level=logging.DEBUG)
+
+
+def get_ot_json_schema():
+    # Temporary schema url for testing
+    schema_url = 'https://raw.githubusercontent.com/opentargets/json_schema/ds_3110_pharmacogenomics_schema/schemas/pharmacogenomics.json'
```

Comment:
Is the schema behind a private repo? I tried accessing it and got a ``404``

Reply:
It's on a branch that's since been merged... I'll update it to point to the main branch, but before the next submission we should update it to point to a tagged version of the schema.

Reply:
A tagged version in case the schema gets changed in ``main``? Would we not be then validating our evidence strings with a version of the schemas that is not the latest for OT?

Reply:
Yes, that's what they request, e.g. in the last email for the December submission window:
> JSON schema: For validating the disease target evidence, please use the latest JSON schema version [2.4.0](https://opentargets.us17.list-manage.com/track/click?u=d11d0467053c1d4b918eb8738&id=81ed75da89&e=e825ffe75c)

This is occasionally a hassle when they make last-minute changes to the schema and then don't make a release in a time, but in theory this shouldn't happen often.

Reply:
(Oh yeah and just to clarify, we update this for every submission so it stays up to date.)

Reply:
Was gonna ask that last bit, I thought it would have been easier for us not to be changing the static releases every submission. 

Might be worth it to check if their GH repo is maintained in a way so that ``main`` is always up to date. If so, and if they always wants us to use the latest version of the schema (which I would expect, because why push it to ``main`` otherwise), a pointer to ``main`` would be less stressful 🤔

Reply:
I think it's more about traceability, since a pointer to `main` will change they presumably want to know exactly which version was used to validate... on the other hand, they also want us to use a specific EFO version and we don't pin that, just use whatever is the latest supported by OLS... so yes, you're probably right that we should double check!